compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -56,6 +56,12 @@ def DispatchLinalgOnTensors :
   let constructor = "mlir::iree_compiler::IREE::Flow::createDispatchLinalgOnTensorsPass()";
 }
 
+def DispatchWithTransformDialect :
+    InterfacePass<"iree-flow-dispatch-linalg-on-tensors-transform-dialect-pass", "mlir::FunctionOpInterface"> {
```

Comment:
we try to keep pass names matching the C++/file names for clarity;
with a filename of DispatchWithTransformDialect this should be `iree-flow-dispatch-with-transform-dialect` (no `-pass`)